### PR TITLE
Jr tabbed interface

### PR DESCRIPTION
**Pull request by jb520**
Created a user navbar with links to liked photos, a list of the user's followers, a list of who the user is following, and the user's feed.

### CODE REVIEW

File: app/controllers/users_controller.rb
Comment (APatel-AI) [inline app/controllers/users_controller.rb:1]: Great job on the users controller.

File: app/views/users/_user_navbar.html.erb
Comment (APatel-AI) [inline app/views/users/_user_navbar.html.erb:1]: navbar partial looks good

**Review by APatel-AI — APPROVED** [review]
Code looks good. It is well structured and concise.

**Review by MayaS1111 — APPROVED** [review]
Great job using the followers and leaders method! All looks good.

**Review by CarlosCastanedaDev — APPROVED** [review]
Nice job creating the navbar partial. You can know merge these changes back to main.

**Pull request merged**